unittests/allplatformstests.py

```diff
@@ -1354,12 +1349,10 @@ def test_lto_mode(self):
 
         env = get_fake_env(testdir, self.builddir, self.prefix)
         cc = detect_c_compiler(env, MachineChoice.HOST)
-        if cc.get_id() != 'clang':
-            raise SkipTest('Only clang currently supports thinLTO')
+        if cc.get_id() not in {'clang', 'clang-cl'}:
+            raise SkipTest('Only clang/clang-cl currently supports thinLTO')
```

Comment:
s/supports/support

Reply:
I don't follow.  Are you saying:  Replace 'supports' with 'support'?  If so, that's not valid.

Reply:
That is what I am saying, and yes it is correct. The string is equivalent to: "Only clang and clang-cl supports thinLTO", which doesn't make sense.

Reply:
Agreed. Think I was reading it wrongly.
Done.